CommunityToolkit.HighPerformance/Streams/Sources/ArrayOwner.cs

```diff
@@ -74,4 +74,14 @@ public Span<byte> Span
 #endif
         }
     }
+
+    /// <inheritdoc/>
+    public Memory<byte> Memory
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        get
+        {
+            return this.array.AsMemory(this.offset, this.length);
+        }
```

Comment:
Nit: use a one liner.
```suggestion
        get => this.array.AsMemory(this.offset, this.length);
```